botcity/maestro/impl/v2.py

```diff
@@ -143,13 +143,13 @@ def create_task(self, activity_label: str, parameters: Dict[str, object],
         """
         url = f'{self._sdk._server}/api/v2/task'
 
-        if min_execution_date is not None and not isinstance(min_execution_date, datetime.datetime):
-            raise ValueError(f"Arg 'min_execution_date' is not datetime. Type {type(min_execution_date)}")
         data = {
             "activityLabel": activity_label, "test": test, "parameters": parameters, "priority": priority,
-            "minExecutionDate": min_execution_date.isoformat()
         }
 
+        if min_execution_date and isinstance(min_execution_date, datetime.datetime):
+            data["minExecutionDate"] = min_execution_date.isoformat()
```

Comment:
We need to add a validation if it is present and in the case it is not a `datetime`. We need to combine what you had before with this.

```python
if min_execution_date is not None:
    if not isinstance(min_execution_date, datetime.datetime):
        raise ValueError(f"Arg 'min_execution_date' is not datetime. Type {type(min_execution_date)}")
    data["minExecutionDate"] = min_execution_date.isoformat()
```